src/tiles/styles.scss

```diff
@@ -63,6 +63,10 @@ $gutter: awsui.$space-grid-gutter;
     background: awsui.$color-background-tiles-disabled;
   }
 
+  &.read-only {
```

Comment:
Why did this change to `read-only` here while all the previous files had it as `readonly`?

Reply:
updated